R/get_publications.R

```diff
@@ -18,13 +19,19 @@ get_publications <- function(
     api_url(
       ees_environment = ees_environment,
       api_version = api_version,
+      search = search,
       page_size = page_size,
       page = page,
       verbose = verbose
     )
   ) |>
     httr::content("text") |>
     jsonlite::fromJSON()
+  if (!is.null(search)) {
```

Comment:
Might be more general for the function as a whole rather than search, but could we add a nice message to return if there were no rows found? (Guessing it might be a lot more common if people are trying to search)

![image](https://github.com/user-attachments/assets/cb14b382-7ac5-4bfb-8776-7b722c78d078)

Reply:
Done:
> `x <- get_publications(search = "dashboardy")`
> Warning message:
> In warning_no_rows(warning_max_pages(response)) :
>  Your query returned zero rows.